core/src/main/java/org/apache/iceberg/util/SortOrderUtil.java

```diff
@@ -64,4 +68,16 @@ static SortOrder buildSortOrder(Schema schema, PartitionSpec spec, SortOrder sor
 
     return builder.build();
   }
+
+  public static Set<String> getSortedColumns(SortOrder sortOrder) {
```

Comment:
nit: `getSortedColumns` -> `sortColumns`. We usually try to avoid `get` in method names. It is a style preference we follow in Iceberg. Such prefixes usually have no meaning and just make the names longer.

Reply:
Yep, this was my first PR to iceberg and I realized this :).  Changed